lib/writer.js

```diff
@@ -180,20 +180,18 @@
       ]);
     }
 
-    function container(type) {
-      return it => {
-        const trivia = extract_trivia(it);
-        const ext = extended_attributes(it.extAttrs);
-        const partial = token(it.partial, "partial");
-        const name = ts.name(it.escapedName, { data: it });
-        const head = wrap`${trivia.base}${type}${trivia.name}${name}`;
-        const inherit = inheritance(it.inheritance);
-        const body = brace_body(trivia, iterate(it.members, it));
-        return ts.definition(
-          wrap`${ext}${partial}${head}${inherit}${body}`,
-          { data: it }
-        );
-      };
+    function container(it) {
```

Comment:
Maybe we can rename  “it” to something more specific?

Reply:
Had no good idea. Do you have a suggestion? `data` would be helpful for `{ data }` but is not any specific.

Reply:
No :( maybe will go through it after the refactor and think of something.